book/release-service.md

```diff
@@ -31,13 +31,13 @@ The Release service is dependent on the following services:
 
 The diagram below shows the interaction of the release service and other services.
 
-![](../diagrams/hacbs-data-flow.jpg)
+![](../diagrams/release-service/hacbs-data-flow.jpg)
```

Comment:
I think this might be appropriate to live outside of specific services. As an example, it is already trying to be referenced by integration-service: https://github.com/redhat-appstudio/book/pull/37/files

Reply:
left in root `/diagrams` folder